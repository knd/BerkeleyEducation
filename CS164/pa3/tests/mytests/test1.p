contains(A,[A|R]).
contains(A,[B|R]) :- contains(A,R).
remove(A,[A|R],R).
remove(A,[F|R],[F|S]) :- remove(A,R,S).
append([A|B],Z,[A|W]) :- append(B,Z,W).
append([],A,A).
remove(A,[1,2,3,4],B)?
