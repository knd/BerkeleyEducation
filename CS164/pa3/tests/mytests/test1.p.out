Asking for solution 1
A = 1
B = [2|[3|[4|[]]]]
Asking for solution 2
A = 2
B = [1|[3|[4|[]]]]
Asking for solution 3
A = 3
B = [1|[2|[4|[]]]]
Asking for solution 4
A = 4
B = [1|[2|[3|[]]]]
Asking for solution 5
None
